pom.xml

```diff
@@ -155,12 +167,6 @@
 		</exclusion>
 	  </exclusions>
     </dependency>
-    <dependency>
-      <groupId>org.hamcrest</groupId>
-      <artifactId>hamcrest-core</artifactId>
-      <version>1.2.1</version>
-      <scope>test</scope>
-    </dependency>
 	<dependency>
```

Comment:
A later version is already included transitively via junit.

As per 4c3cc141867c62655c0b451044a4d3c54f2e5bd8:
"Rely on later version required by JUnit. The previous configuration caused Eclipse/m2e compilation path issues."